l10n_it_fatturapa_in/wizard/link_to_existing_invoice.xml

```diff
@@ -10,7 +10,7 @@
                     <group>
                         <group>
                             <field name="invoice_id"
-                                   domain="[('type', 'in', ('in_invoice', 'in_refund')), ('state', 'in', ('draft', 'proforma', 'proforma2')), ('fatturapa_attachment_in_id', '=', False)]"
+                                   domain="[('type', 'in', ('in_invoice', 'in_refund')), ('state', '!=', 'cancel'), ('fatturapa_attachment_in_id', '=', False)]"
```

Comment:
@sergiocorato in quali casi potresti voler collegare una fattura elettronica ad una fattura odoo già aperta o pagata? In altri termini: perchè dovresti avere una fattura odoo aperta o pagata, se la fattura elettronica non l'hai ancora ricevuta?

Reply:
@eLBati perchè l'ho generata tramite Odoo nelle varie maniere automatiche possibili

Reply:
Se l'hai generata ce l'hai in bozza. Poi l'hai anche validata senza averla ricevuta?

Reply:
Io no, ma utenti che ricevono magari il cartaceo o il pdf e vanno avanti lo stesso purtroppo ce ne sono

Reply:
Ok allora